lib/snmplib/mib.c

```diff
@@ -260,10 +257,7 @@ int *out_len;       /* number of subid's in "output" */
     return (1);
 }
 
-void
-print_objid(objid, objidlen)
-oid *objid;
-int objidlen;       /* number of subidentifiers */
+void print_objid(oid *objid, int objidlen)
```

Comment:
Restoring formatting and lost comment:

```suggestion
/// \param objidlen number of subidentifiers
void
print_objid(oid *objid, int objidlen)
```